src/utils/internal/getCallFrame.test.ts

```diff
@@ -0,0 +1,163 @@
+import { getCallFrame } from './getCallFrame'
+
+const mockStack = jest.fn()
+jest.spyOn(window, 'Error').mockImplementation(() => ({
```

Comment:
Please put this spying in the `beforeAll` hook. Don't forget to add the `afterAll` hook that `jest.restoreAllMocks()` clears things out.

Reply:
I thought that Jest spies work on the level of a single file, but can't any documentation whether that's true or not :(

- [ ] add explicit beforeAll/afterAll